src/img2catalog/xnat_parser.py

```diff
@@ -217,12 +217,12 @@ def xnat_to_FDP(
 
     for dataset, subject in tqdm(dataset_list):
         dataset_graph = dataset.to_graph(subject)
-        prepare_dataset_graph_for_fdp(dataset_graph, catalog_uri)
+        dataset_graph.add((URIRef(dataset.identifier), DCTERMS.isPartOf, catalog_uri))
```

Comment:
You left the old function intact, so I'd keep calling that one. It's more clear on what it does.